raddb/mods-available/rest

```diff
@@ -53,6 +53,12 @@ rest {
 	#
 #	connect_timeout = 4.0
 
+	#
+	# Specify HTTP protocol version to use. one of '1.0', '1.1', '2.0', '2.0+auto',
+	# '2.0+tls' or 'default'. (libcurl option CURLOPT_HTTP_VERSION)
+	#
+#	http_negotiation = 1.1
```

Comment:
This isn't enough to support HTTP2, there's also some header parser changes required.

https://github.com/FreeRADIUS/freeradius-server/commit/6286520698a3cc4053b4d49eb0a61d9ba77632aa#diff-0cca2197649953eab611e29cf27fdcc98624c3b46bed73725b4294674c3fcb44

Some of the other changes in the commit series may also be required, you need to _test_ the code against a HTTP2.0 server to be sure.

Reply:
yep, the OKTA service supports HTTP 1.0, 1.1 and 2.0 as well.